AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/AppLovin/MRECs/ALDemoProgrammaticMRecViewController.swift

```diff
@@ -27,16 +28,23 @@ class ALDemoProgrammaticMRecViewController : ALBaseAdViewController
         adView.adDisplayDelegate = self
         adView.adEventDelegate = self
         adView.translatesAutoresizingMaskIntoConstraints = false
+        callbackTableView.translatesAutoresizingMaskIntoConstraints = false
         
         // Call loadNextAd() to start showing ads
         adView.loadNextAd()
         
         view.addSubview(adView)
+        guard let callbackTableView = callbackTableView else { return }
```

Comment:
Not sure this `nil` check is necessary here since it's an `@IBOutlet`, where the IB holds a reference to the view. We should be concerned with null-ability if it was an Optional.

Reply:
my xcode is throwing warnings regarding the null-ability of this outlet. I can take it out for this PR

Reply:
Isn't it common practice to mark IBOuetlets with `!` to guarantee its existence?

Reply:
Strange thing is, it is marked with a !     
```
@IBOutlet weak var callbackTableView: UITableView!
```